skimage/filters/thresholding.py

```diff
@@ -1022,3 +1024,88 @@ def apply_hysteresis_threshold(image, low, high):
     connected_to_high = sums > 0
     thresholded = connected_to_high[labels_low]
     return thresholded
+
+
+def threshold_multiotsu(image, classes=3, nbins=256):
+    """Generates multiple thresholds for an input image, chosen to maximize the
```

Comment:
```suggestion
    """Generates multiple thresholds for an input image.
    
    These thresholds are chosen to maximize the
```

A one line short summary is preferred.

Reply:
I inserted the rest of the sentence in the Notes section.